modules/rostests/apitests/gdi32/CreateDIBitmap.c

```diff
@@ -334,7 +334,7 @@ Test_CreateDIBitmap1(void)
     ok(bitmap.bmType == 0, "\n");
     ok(bitmap.bmWidth == 2, "\n");
     ok(bitmap.bmHeight == 2, "\n");
-    ok(bitmap.bmWidthBytes == 8, "bmWidthBytes = %ld\n", bitmap.bmWidthBytes);
+    ok(bitmap.bmWidthBytes == 4, "bmWidthBytes = %ld\n", bitmap.bmWidthBytes);
```

Comment:
Should this be `bitmap.bmWidth * 8 *  bitmap.bmBitsPixel` or some variation thereof?

Reply:
Should be something like ```ALIGN_UP_BY(bitmap.bmWidth * bitmap.bmBitsPixel, 16) / 8``` I'll fix that.